packages/twenty-server/src/engine/twenty-orm/custom.workspace-entity.ts

```diff
@@ -150,5 +150,5 @@ export class CustomWorkspaceEntity extends BaseWorkspaceEntity {
   @WorkspaceIsNullable()
   @WorkspaceIsSystem()
   @WorkspaceFieldIndex({ indexType: IndexType.GIN })
-  [SEARCH_VECTOR_FIELD.name]: any;
```

Comment:
Seems like there's an issue with the lingui parser which unfortunately forced me to rename all the fields with a dynamic variable name like that. We should probably open an issue on Lingui's repo